testkit/src/main/scala/org/bitcoins/testkit/EmbeddedPg.scala

```diff
@@ -0,0 +1,52 @@
+package org.bitcoins.testkit
+
+import com.opentable.db.postgres.embedded.EmbeddedPostgres
+import org.bitcoins.testkit.BitcoinSTestAppConfig.ProjectType
+import org.scalatest.{BeforeAndAfterAll, Suite}
+
+import scala.util.Try
+
+trait EmbeddedPg extends BeforeAndAfterAll { this: Suite =>
+
+  val pgEnabled: Boolean = !sys.env.contains("NO_PG")
```

Comment:
If I understand this correctly, we are defaulting to use postgres for all test cases. I think we should default to postgres off. I also think we should rename this env variable to `PG_ENABLED`